fab/fabfile.py

```diff
@@ -728,6 +730,18 @@ def force_update_static():
     execute(services_restart)
 
 
+def _tag_commit():
+    sh.git.fetch("origin", env.code_branch)
```

Comment:
I hardcoded `"origin"` as the name of the remote branch, which I realize might not always be the case.  Is there a better way to tell?